azure-pipelines-nightly.yaml

```diff
@@ -0,0 +1,143 @@
+name: VH Shared Services AKS Azure Infrastructure Nightly Drift Check Pipeline
+trigger:
+  - none
+
+pr:
+  - none
+
+schedules:
+- cron: "0 21 * * 1,2,3,4,5"
+  displayName: Nightly
+  branches:
+    include:
+    - master
+  always: true
+- cron: "0 21 1 * *"
+  displayName: Monthly
+  branches:
+    include:
+    - master
+  always: true
+
+
+resources:
+  repositories:
+    - repository: templates
+      type: github
+      name: hmcts/azure-devops-templates
+      ref: refs/heads/master
+      endpoint: hmcts
+      
+parameters:
+  - name: Run_Stage
+    displayName: Stage to Run
+    type: string
+    default: 'Check_Drift'
+    values:
+      - Check_Drift
+
+  - name: location
+    displayName: Location
+    type: string
+    default: 'UK South'
+    values:
+    - 'UK South'
+    - 'UK West'
+
+  - name: env
+    displayName: Environment
+    type: string
+    default: 'SBOX'
+    values:
```

Comment:
I'd EITHER remove the optional values or change the name of this pipeline to azure-pipelines-drift-check.yaml